src/e2e-test/java/io/cdap/plugin/BQValidation.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin;
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.Field;
+import com.google.cloud.bigquery.FieldList;
+import com.google.cloud.bigquery.LegacySQLTypeName;
+import com.google.cloud.bigquery.Schema;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TableResult;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.utils.SalesforceClient;
+import io.cdap.plugin.utils.enums.SObjects;
+import org.apache.http.Header;
+import org.apache.http.HttpHeaders;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.message.BasicHeader;
+import org.apache.http.util.EntityUtils;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.text.ParseException;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.sql.*;
+
+/**
+ * BQValidation.
+ */
+public class BQValidation {
+
+  static List<JsonObject> bigQueryResponse = new ArrayList<>();
+  static List<Object> bigQueryRows = new ArrayList<>();
+  static Gson gson = new Gson();
+
+  public static void main(String[] args) throws IOException, InterruptedException, SQLException, ParseException {
+//    List<Object> l=new ArrayList<>();
+//
+//    getBigQueryTableData("E2E_SOURCE_7ff9f7fc_bf6c_46d1_a050_1d4525e7b575",l);
+//    for(int i=0;i<l.size();i++)
+//    {
+//      System.out.println(l.get(i));
```

Comment:
Please remove the main function from here.

Reply:
done